eclipse/src/saros/ui/widgets/viewer/roster/ContactSelectionComposite.java

```diff
@@ -202,23 +194,21 @@ public void setSelectedContacts(List<JID> contacts) {
    *
    * @param allRosterEntryElements
    * @param checkedRosterEntryElement {@link RosterEntryElement}s which are already checked
-   * @param rosterEntryElementToCheck {@link RosterEntryElement}s which have to be exclusively
-   *     checked
+   * @param contacts {@link List} of {@link JID JIDs} which have to be exclusively checked
    * @return {@link Map} of {@link RosterEntryElement} that must change their check state
    */
   protected Map<RosterEntryElement, Boolean> calculateCheckStateDiff(
       List<RosterEntryElement> allRosterEntryElements,
       List<RosterEntryElement> checkedRosterEntryElement,
-      List<RosterEntryElement> rosterEntryElementToCheck) {
+      List<JID> contacts) {
```

Comment:
Beside how this stuff is done, why you changed it ?

Reply:
I just done it to avoid `new RosterEntryElement` in L170 (i would need contact objects)